pages/_news/2022-04-22-TrainingMaterial-1.0-RELEASE.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+title: "TrainingMaterial Profile 1.0-RELEASE Published"
+tags:
+- Profile
+- TrainingMaterial
+---
+We are pleased to announce the release of the [TrainingMaterial Profile](/profiles/TrainingMaterial/1.0-RELEASE) from the [Training Working Group](https://bioschemas.org/groups/Training).
```

Comment:
@ppalagi has provided the following text, please update accordingly

We are pleased to announce the long-awaited release of the [TrainingMaterials](https://bioschemas.org/profiles/TrainingMaterial/1.0-RELEASE) profile from the [Training Working Group](https://bioschemas.org/groups/Training).

This release represents an important first step in profiles from the Training Working Group, with further releases planned soon for [Course](https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08) and [CourseInstance](https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06) profiles.

Example deployments of the TrainingMaterials profile can be found at [ZBMED-SEMTEC](https://zbmed-semtec.github.io/BioMedSem-IR-KD/) and [enanomapper](https://enanomapper.github.io/tutorials/Entering_and_analysing_nano_safety_data/readme.html), as well as [Bioschemas.org](https://bioschemas.org/tutorials/what_why_bioschemas) and [nanocommons](https://nanocommons.github.io/tutorials/enteringData/).

Reply:
Fixed in [f70f21a](https://github.com/BioSchemas/bioschemas.github.io/pull/527/commits/f70f21aa77c178469e4ebd426a8abf4e6da4bfea)